front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/v2/EntityTagsController.groovy

```diff
@@ -91,6 +90,15 @@ class EntityTagsController {
     return findAllByIds(tags.findResults { it.id })
   }
 
+  @RequestMapping(value = "/batchDelete", method = RequestMethod.POST)
```

Comment:
`/batchDelete` seems funny given that the below the fold api is `bulkDelete` ... but we already have a `/batchUpdate` endpoint and I figured it was better to adhere to that pattern.

Internal APIs anyhow.